pkgs/development/tools/continuous-integration/woodpecker/common.nix

```diff
@@ -1,7 +1,7 @@
 { lib, fetchFromGitHub }:
 let
-  version = "1.0.1";
-  srcHash = "sha256-uwKLD3fW/em6UMkkyrWxAo7T//Hkzj6WjIp5qJVtBuc=";
+  version = "1.0.2";
+  srcHash = "sha256-OeOKR9lTGXU2qumyXq3s/CI56IX4XiI/ZTRHNoY6MEI=";
   vendorHash = "sha256-NYWJorVeRxbQTiirHK8gqpDddn2RsKsNWwDNdcONVQA=";
```

Comment:
go.mod changed in this release, so vendorHash needs to be updated.

```
┃          specified: sha256-NYWJorVeRxbQTiirHK8gqpDddn2RsKsNWwDNdcONVQA=
┃             got:    sha256-+2wQKNyCb9xtB1TeE1/oSMEvKoXVX9ARZxsNqE2rfrg=
```

Reply:
Not sure how that was missed, as I used the update script 🤔.

Reply:
That's already the case ;-)

Reply:
Oops, I grep'd the wrong location. 🙈